code/datums/uplink_item.dm

```diff
@@ -952,6 +952,15 @@ var/list/uplink_items = list()
 	cost = 12
 	jobs_exclusive = list("Mime")
 
+/datum/uplink_item/jobspecific/clown_mime/punchline
+	name = "Punchline"
+	desc = "A high risk high reward abomination combining experimental phazon and bananium technologies. Wind-up Punchline to charge it. Enough charge and your targets will slip through reality. Warning: Forcing wind-ups beyond the limiter may reverse the prototype phazite honkpacitors and disrupt reality around the user."
+	item = /obj/item/weapon/gun/hookshot/whip/clownbox
+	cost = 14
+	discounted_cost = 10
+	jobs_with_discount = list("Clown")
+	jobs_excluded = list("Mime")
```

Comment:
why? no other traitor item uses this currently

Reply:
Mimes do not tell jokes.

Reply:
There's a lot of clown/mime only items so I didn't want to limit it to just clown. It's very un-mime though. It screams bike horns all over the place when it malfunctions, etc. It would be weird to be used by a mime.

Reply:
I don't get why it makes sense for mimes to buy a banana grenade and banana gun but not this

Or why it makes more sense for the chaplain or head of security or whatever to get this

Reply:
The former mostly doesn't make sense. Though at least bananas are quiet. 
The latter also doesn't make sense but it's more fun. Clowns already get a lot of benefit vs anyone else. Not only is it a lot cheaper for them but they reload it much faster. Really I'd say it isn't a worthwhile item for anyone but clowns, I just didn't want to limit it because fun. 
The mime exclusion is specifically because it's very loud. It makes horn sounds when you wind it, party favor sounds when you fire it. If a mime uses it and it malfunctions at high springForce it should warrant a vow break.